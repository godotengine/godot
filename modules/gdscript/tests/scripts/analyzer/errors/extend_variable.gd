# GH-75870

var A = 1

class B extends A:
	pass

func test():
	pass
