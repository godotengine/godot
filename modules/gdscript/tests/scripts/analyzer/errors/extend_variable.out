GDTEST_ANALYZER_ERROR
>> ERROR at line 5: Cannot use variable "A" in extends chain.
